application/log_handlers/handlers.py

```
@@ -0,0 +1,181 @@
+            return env_uuid.strip()[:8]  # Limit to first 8 characters
+
+        # 2️⃣ Define a persistent file path
+        if os.name == "nt":  # Windows
+            uuid_file = Path(os.getenv("APPDATA", "C:/ProgramData")) / "iCtrl_machine_log_uuid.txt"
+        else:  # Linux/macOS
+            uuid_file = Path(
+                "/etc/iCtrl_machine_log_uuid") if os.geteuid() == 0 else Path.home() / ".iCtrl_machine_log_uuid"
+
+        # 3️⃣ Check if the file exists and read the UUID
+        if uuid_file.exists():
+            with open(uuid_file, "r") as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)